data/geoscience/geochronology/index.html

```diff
@@ -33,6 +45,30 @@
     item_id: 3c3d19ed6a4c43ec8fb5d02d1dde37d4
   updates:
     - August 2020
+RubidiumStrontium:
+  hub:
+    name: Utah Geochronology Rubidium Strontium
```

Comment:
**issue**: this isn't the correct name or something is up since it is creating a 404 in the links.